README.md

```diff
@@ -1,5 +1,5 @@
 # Microsoft REST API Guidelines
-The [Microsoft REST API Guidelines](Guidelines.md), as a design principle, encourages application developers to have resources accessible to them via a RESTful HTTP interface. To provide the smoothest possible experience for developers on platforms following the Microsoft REST API Guidelines, REST APIs SHOULD follow consistent design guidelines to make using them easy and intuitive.
+The [Microsoft REST API Guidelines](Guidelines.md), as a design principle, encourages application developers to have resources accessible to them via a RESTful HTTP interface. To provide the smoothest possible experience for developers on platforms following the Microsoft REST API Guidelines, REST APIs SHOULD follow [consistent design guidelines](https://github.com/IAmHughes/api-guidelines/blob/vNext/Guidelines.md#7-consistency-fundamentals) to make using them easy and intuitive.
```

Comment:
Relative link? Or am I misunderstanding how GitHub handles links?

Reply:
yes this is a relative link to the header within the `Guidelines.md` doc.

Reply:
So weird - I get to see it as an absolute URL to your fork of the repo.

Reply:
I merged it and it still was absolute so I hand fixed it after the fact to be relative.

Reply:
Weird! Sorry about that @garethj-MSFT, thank you!